src/js/component.js

```diff
@@ -369,6 +369,10 @@ class Component {
       // If there's no .player_, this is a player
       let ComponentClass = Component.getComponent(componentClassName);
 
+      if (!ComponentClass) {
```

Comment:
Elaborating a bit-- if a bad component was encountered midway through player creation, this would abort early and return a player missing a chunk of itself. It seems like a bad component name is a development-time-misconfiguration and it might be better to fail fast and force a fix.